_docs/collection-details.md

```diff
@@ -30,6 +30,35 @@ To initiate a GHS mobile collection, please use the following details (phone_num
 
 {% include language-tabbar.html prefix="collection-ghs-mobile" raw=data-raw %}
 
+Once the transaction has been created the `out_details` will be:
```

Comment:
`will be` implies that this will always be the case, while it's only applicable to MTN. Is this what we want?

Reply:
Correct Evanson, thanks for the spot, updating it now